### PR TITLE
Adjust operations and certs diagrams

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Remove the outdated "update" diagram
- Update "provisioning", "deprovisioning", and "certs" diagrams
- Add the "release" diagram created in diagram.net

**Related issue(s)**
#353

### CODE REVIEW

File: docs/assets/deprovisioning.svg
Comment (nataliasitko) [inline docs/assets/deprovisioning.svg:1]: [Our guidelines suggest](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/09-diagrams.md#background) using mint green (HEX: #DEF2DD) for subsidiary environments. Would it make sense to replace those white "Hard delete" and "Soft delete" rectangles with rounded rectangles in that mint green color?

Comment (nataliasitko) [reply docs/assets/deprovisioning.svg:1]: Also, I don't know if we should use sentence case for the texts in diagrams or begin with a lowercase letter. It seems that we're not consistent about that in Kyma

Comment (IwonaLanger) [reply docs/assets/deprovisioning.svg:1]: > [Our guidelines suggest](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/09-diagrams.md#background) using mint green (HEX: #DEF2DD) for subsidiary environments. Would it make sense to replace those white "Hard delete" and "Soft delete" rectangles with rounded rectangles in that mint green color?

Now that you mention it, it makes perfect sense :) I'll change the color.

Comment (IwonaLanger) [reply docs/assets/deprovisioning.svg:1]: > Also, I don't know if we should use sentence case for texts in diagrams or begin with a lowercase letter. It seems that we're not consistent about that within Kyma

Same thoughts exactly. I'll have another look and adapt the prevailing approach.

**Review by nataliasitko — APPROVED** [review]

File: docs/assets/deprovisioning.svg
Comment (IwonaLanger) [reply docs/assets/deprovisioning.svg:1]: > Thanks, I can create an issue to update the diagrams that use the less common approach :)

Good idea :)

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**